basics/imports-and-modules.md

```diff
@@ -0,0 +1,33 @@
+# Import och moduler
+
+För ett simpelt hej världen program i D behövs `import` satsen.
+`import` satsen gör alla publika funktioner samt typer
+ifrån den angivna **modulen** tillgängliga.
+
+Standard biblioteket vid namn [Phobos](https://dlang.org/phobos/)
+finns under  **paketet** `std`
+och dess moduler hänvisas till genom `import std.MODUL`.
+
+`import` satsen kan också användas selektivt
+för att importera specifika delar av en modul:
+
+    import std.stdio : writeln, writefln;
+
+Selektiva imports kan också användas för att förbättra läsbarheten
+genom att göra det uppenbart varifrån en symbol kommer ifrån,
+det är också ett sätt att förhindra konflikter mellan symboler som har
+samma namn men inte tillhör samma modul.
+
+En `import` sats behöver inte befinnas i början av källflen.
+Den kan också användas lokalt inom funktioner.
```

Comment:
You missed the other half of that sentence.
`[...] eller inom någon annan omfattning.`

I don't know what `scope` translates best to but `omfattning` seems the best. idk